db/migrate/20190413205758_create_app_migrations.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAppMigrations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :app_migrations, id: :uuid, default: -> { "uuid_generate_v4()" } do |t|
+      t.references :app, type: :uuid, null: :false, foreign_key: true
+      t.string :version, null: false, unique: true
```

Comment:
`unique: true` doesn't appear to be doing anything. is that okay for you?

Reply:
seems like you don't need it here since you have the specific index below.

Reply:
@jmileham might just wanna remove this to avoid confusion

Reply:
If you wanted it to work, would it be `index: { unique: true }`?

Reply:
yeah, i will remove - it is too tight, wouldn't allow multiple apps to have migrations with the same timestamp, which is unnecessary.

Reply:
```suggestion
      t.string :version, null: false
```